t/plugin/key-auth.t

```diff
@@ -23,6 +23,31 @@ use t::APISIX 'no_plan';
 repeat_each(2);
 no_long_string();
 no_root_location();
+
+add_block_preprocessor(sub {
+    my ($block) = @_;
+
+    my $user_yaml_config = <<_EOC_;
+deployment:
+  role: traditional
+  role_traditional:
+    config_provider: etcd
+  admin:
+    admin_key: null
+apisix:
+  node_listen: 1984
+  proxy_mode: http&stream
+  stream_proxy:
+    tcp:
+      - 9100
+  enable_resolv_search_opt: false
+  data_encryption:
+    enable_encrypt_fields: false
```

Comment:
disable encrypt fields for this test file because this line in the tests:
https://github.com/shreemaan-abhishek/apisix/blob/7c2994b844aef288030392d85bcc9bfbfb46ad0a/t/plugin/key-auth.t#L206-L207

expects response with sensitive fields in plain text... modifying the test to expect encrypted fields would make the tests too complex.

Reply:
Could we only keep 

```
apisix:
    data_encryption:
        enable_encrypt_fields: false
```
this ?

Reply:
I think this should work. let me try